api/simplestore/sessions_test.go

```diff
@@ -0,0 +1,121 @@
+package simplestore
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/18F/e-QIP-prototype/api"
+	"github.com/google/uuid"
+)
+
+func getDateAndUUID() (string, time.Time) {
+	return uuid.New().String(), time.Now().Add(time.Duration(5 * time.Minute))
+}
+
+func getTestObjects(t *testing.T) (ss SimpleStore, account api.Account, UUID string, date time.Time) {
```

Comment:
We don't use named returns anywhere in this codebase. I think I'd prefer we keep it that way rather than having to watch out for them sometimes.

Reply:
:+1: since `getUUIDAndDate` is hard to read, updating the order & calling code.